browser/ui/BUILD.gn

```diff
@@ -145,6 +145,8 @@ source_set("ui") {
     sources += [
       "android/ai_chat/brave_leo_settings_launcher_helper.cc",
       "android/ai_chat/brave_leo_settings_launcher_helper.h",
+      "android/rewards/rewards_page_helper.cc",
+      "android/rewards/rewards_page_helper.h",
```

Comment:
Can we name this folder `brave_rewards` to match the other folder names?

Also, would these file names be more descriptive with something like `rewards_page_activity_helper`?